go/vt/sqlparser/ast_format.go

```diff
@@ -1353,6 +1353,66 @@ func (node *ExtractFuncExpr) Format(buf *TrackedBuffer) {
 	buf.astPrintf(node, "extract(%s from %v)", node.IntervalTypes.ToString(), node.Expr)
 }
 
+// Format formats the node
+func (node *RegexpInstrExpr) Format(buf *TrackedBuffer) {
+	buf.astPrintf(node, "regexp_instr(%v, %v", node.Expr, node.Pattern)
+	if node.Position != nil {
+		buf.astPrintf(node, ", %v", node.Position)
+	}
+	if node.Occurrence != nil {
+		buf.astPrintf(node, ", %v", node.Occurrence)
+	}
+	if node.ReturnOption != nil {
+		buf.astPrintf(node, ", %v", node.ReturnOption)
+	}
+	if node.MatchType != nil {
+		buf.astPrintf(node, ", %v", node.MatchType)
+	}
+	buf.WriteByte(')')
+}
+
+// Format formats the node
+func (node *RegexpLikeExpr) Format(buf *TrackedBuffer) {
+	if node.IsNot {
+		buf.WriteString("not ")
```

Comment:
I think we also want `buf.astPrintf` here to have consistent casing of `NOT` also for the canonical format.

Reply:
The field `IsNot` was added to represent `Regexp` and `Rlike` in terms of `REGEXP_LIKE` when the syntax looked something similar to : `expr NOT REGEXP expr`. 
As we parse this differently now itself, this is redundant and won't be used anywhere.
I will remove this field.

Reply:
@K-Kumar-01 Ah, if it's redundant, even better to remove it!